docs/pubsub-usage.rst

```diff
@@ -20,24 +32,27 @@ Create a new topic for the default project:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> topic = Topic('topic_name')
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = client.topic('topic_name')
```

Comment:
I know it breaks the "one and only one way to do something" but is it crazy to offer the `create_topic` API method in addition?

I'm thinking...

```python
class Client(...):
  def create_topic(self, name):
    topic = self.topic(name)
    topic.create()
    return topic
```

This was partly due to the (what I believe correct) stance that `.create()` shouldn't return the topic, however if I'm in a REPL, I can see `client.create_topic()` being a very nice-to-have thing.